### PR TITLE
feat(input): DLT-2080 add type search to input

### PR DESCRIPTION
**Pull request by ninamarina**
# Add type "search" to Input

## Obligatory GIF (super important!)

Obligatory GIF

## :hammer_and_wrench: Type Of Change

These types will increment the version number on release:

- [ ] Fix
- [x] Feature
- [ ] Performance Improvement
- [ ] Refactor

## :book: Jira Ticket
[DLT-2080]

## :book: Description
Adds type "search" to input and removes default styling.

## :pencil: Checklist

For all PRs:

- [x] I have ensured no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).
- [x] I have reviewed my changes.
- [x] I have added all relevant documentation.
- [x] I have considered the performance impact of my change.

For all Vue changes:

- [ ] I have added / updated unit tests.
- [x] I have made my changes in Vue 2 and Vue 3. Note: you may sync your changes from Vue 2 to Vue 3 (or vice versa) using the `./scripts/dialtone-vue-sync.sh` script. Read docs here: [Dialtone Vue Sync Script](../packages/dialtone-vue3/.github/CONTRIBUTING.md#dialtone-vue-sync-script)
- [ ] I have validated components with a screen reader.
- [ ] I have validated components keyboard navigation.

[DLT-2080]: https://dialpad.atlassian.net/browse/DLT-2080?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by braddialpad — APPROVED** [review]
Looks good!

File: apps/dialtone-documentation/docs/components/input.md
Comment (braddialpad) [inline apps/dialtone-documentation/docs/components/input.md:1]: Oh, just realized the actual example is not showing a clear button in the input. We probably want to have that right?

**Review by braddialpad — CHANGES_REQUESTED** [review]

File: apps/dialtone-documentation/docs/components/input.md
Comment (ninamarina) [reply apps/dialtone-documentation/docs/components/input.md:1]: The button appears when there is text on the input 😅

Comment (braddialpad) [reply apps/dialtone-documentation/docs/components/input.md:1]: hmm let's maybe populate the search with some text by default then.

**Review by braddialpad — APPROVED** [review]
Just populate search input with text and we're good!

**Pull request merged**